### PR TITLE
fix: add zaira

### PR DESCRIPTION
**Pull request by zairahira**
resolves [#1110](https://github.com/EddieHubCommunity/awesome-github-profiles/issues/1110)

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/zaira.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/33151350/167124706-4332b930-2973-4486-aa00-9c9305ac64da.png",
+  "issueId": 66,
+  "name": "Eddie Jaoude",
+  "username": "eddiejaoude"
```
Comment (vikasganiga05) [inline profiles/zaira.json:5]:
```suggestion
  "issueId": 1110,
  "name": "Zaira Hira",
  "username": "zairahira"
```
Forgot to change it to your credentials.

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Hi, the `issueId` is wrong and you forgot to change `name` & `username` with yours.

**Comment by zairahira** [comment]
@vikasganiga05, thank you for suggesting the change, I missed that.

**Review by Panquesito7 — APPROVED** [review]

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
The file name should be `your-github-username.json`. Rename `zaira.json` to `zairahira.json`.

**Review by vikasganiga05 — APPROVED** [review]
LGTM :+1: Thanks for contributing.

**Review by krshkodes — APPROVED** [review]
Thanks for the contribution! Looks awesome!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient, Elegant, Maintainable

**Review by Pradumnasaraf — APPROVED** [review]
Look good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**